### PR TITLE
Add dolar-cripto-ar extension

### PR DESCRIPTION
**Pull request by koaladlt**
## Description

This extension allows you to view real-time quotes of various Argentine dollars (CCL, MEP, and Blue) and popular cryptocurrencies (ETH, BTC)

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/dolar-cripto-ar/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      crypto: [{ name: "BTC" }, { name: "ETH" }],
+      isLoading: false,
+    });
+  }, []);
+
+  return state;
+};
+
+export default function Command() {
+  const [selectedCurrency, setSelectedCurrency] = useState<string | undefined>();
```
Comment (pernielsentikaer) [inline extensions/dolar-cripto-ar/src/index.tsx:33]: You might want to use [useCachedState](https://developers.raycast.com/utilities/react-hooks/usecachedstate) from `@raycast/utils` instead (and set the default value directly

File: extensions/dolar-cripto-ar/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  const { dollar, crypto } = useRates();
+
+  const { data, isLoading: isFetching } = useFetch<DollarResponse>("https://criptoya.com/api/dolar");
+  const { data: btcData, isLoading: isBtcFetching } = useFetch<{ USD: number }>(
+    "https://min-api.cryptocompare.com/data/price?fsym=BTC&tsyms=USD",
+  );
+  const { data: ethData, isLoading: isEthFetching } = useFetch<{ USD: number }>(
+    "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD",
+  );
```
Comment (pernielsentikaer) [inline extensions/dolar-cripto-ar/src/index.tsx:42]: You can maybe use [useCachedPromise](https://developers.raycast.com/utilities/react-hooks/usecachedpromise) instead

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Can the menubar be redrawn after switching to another currency?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by koaladlt** [comment]
@pernielsentikaer Thanks for the feedback! I've made the changes that you asked for.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
If you clear Local Storage and try to use the menubar, then it flickers with the text "blue" and the current, can you look into that?

Raycast 2024-06-17 at 09 23 54

https://github.com/raycast/extensions/assets/38033107/a9537537-0b2f-46ea-8d46-959ca32a437b

**Comment by koaladlt** [comment]
> If you clear Local Storage and try to use the menubar, then it flickers with the text "blue" and the current, can you look into that?
> 
> Raycast 2024-06-17 at 09 23 54
> 
>  Raycast17062024-4CIhBKll.png.mp4

It should be working fine now! thanks for the feedback.

**Comment by mil3na** [comment]
Hey @koaladlt ! Is this ready for another review?

**Comment by koaladlt** [comment]
> Hey @koaladlt ! Is this ready for another review?

Yes it is :)

**Comment by koaladlt** [comment]
> Thank you so much for your contribution, again! After testing it out, I was missing some kind of indication of which unit is "active". Have you considered adding a check mark (or any kind of mark) on the list next to the unit which is showing at the moment?

Thanks for your feedback! :) I added a check mark to the select unit so the user knows which one is selected.

**Review by mil3na — APPROVED** [review]
Awesome! Thanks again for your contribution!

**Pull request merged**